vaadin-time-picker-flow-parent/vaadin-time-picker-flow-demo/src/main/java/com/vaadin/flow/component/timepicker/demo/TimePickerView.java

```diff
@@ -291,6 +291,7 @@ public void setLocalTime(LocalTime localTime) {
 
         public void setStep(Duration step) {
             this.step = step;
+            updateValue();
```

Comment:
This is part of a component that is used in test setups to verify the time formats for different locales. The component was relying on the fact that changing the step property would trigger a value change in the TimePicker Flow component, which in turn lead to the `updateValue` method being called.

Now changing the step will keep the old value in the component, so it was necessary to trigger `updateValue` additionally here.